NatureArea.py

```diff
@@ -0,0 +1,245 @@
+from WikidataItem import WikidataItem
+import importer_utils as utils
+
+
+class NatureArea(WikidataItem):
+    """
+    Extension of WikidataItem to create nature area objects.
+
+    Handles both reserves and national parks.
+    """
+
+    def generate_ref_url(self):
+        url = "http://nvpub.vic-metria.nu/naturvardsregistret/rest/omrade/{}/G%C3%A4llande"
+        return url.format(self.raw_data["NVRID"])
+
+    def create_sources(self):
```

Comment:
Yes. But any item that calls this will always be of one type or the other. Hence it would only ever have one source.
Right now each "nr" also contains an "np" source (which will never get used but [requires a later check](https://github.com/Vesihiisi/WLE_import/pull/2/files/264941a7e9e5273e427ce001207d6ee739b745d3..66492eec7e7716a63028aa7be3ee7f3ca01fe43e#r114551647) to chose which to use).